css/style.css

```
@@ -1,19 +1,506 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* Header */
+.header-section {
+  display: flex;
+  flex-flow: row nowrap;
+  width: 100%;
+  padding: 1rem 0 1rem 0;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+/* Navigation */
+.company-logo {
+  padding-left: 0.8rem;
+  width: 2.3rem;
+  height: 2.3rem;
+}
+
+.hamburger-icon {
+  padding: 0.8rem 0.8rem 0 0;
+  width: 1.5rem;
+  height: 1.1rem;
+}
+
+.navbar {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.navbar-hamburger-Menu {
+  display: flex;
+  flex-flow: column nowrap;
+  background-color: #fff;
+  border: 0;
+}
+
+.navbar-full-Menu.show {
+  display: none;
+}
+
+.navbar-full-Menu {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.navbar-full-Menu > li {
+  list-style-type: none;
+}
+
+.navbar-full-Menu > li:first-child {
+  color: #4b4b4d;
+  font-weight: 500;
+}
+
+.navbar-full-Menu > li > a {
+  line-height: 3rem;
+  color: #89899e;
+  text-decoration: none;
+}
+
+.navbar-full-Menu > li > a:hover {
+  color: #e0633a;
+  cursor: pointer;
+}
+
+/* Hero */
+.hero-section {
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  background: url(../img/first-background.jpg) 67%;
+  background-size: cover;
+  height: 26.5rem;
+}
+
+.hero-section > h1 {
+  font-weight: 100;
+  font-size: 2.5rem;
+  margin: 0.3rem 6rem 0rem 6rem;
+}
+
+.hero-section > p {
+  font-weight: 100;
+  font-size: 1.4rem;
+  margin: 0rem 5rem 0rem 5rem;
+  padding-top: 2rem;
+}
+
+.hero-section > button {
+  background-color: #e0633a;
+  border: 0px;
+  border-radius: 4px;
+  margin-top: 2rem;
+  width: 7.4rem;
+  height: 2.8rem;
```

```suggestion
  padding: 1rem;
```
Rather than set a specific width and height, we can use padding. The benefit is that the width sizes nicely to the text content whether there's a little or a lot of text.